signing/src/main/java/tech/pegasys/web3signer/signing/metrics/FilecoinMetricCategory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 ConsenSys AG.
```

Comment:
Should this be 2022? Was it not generated via spotless?

Reply:
Yeah good point, it should be 2022 since it's new code. I'll fix that up

Reply:
done

Reply:
done